devicetypes/smartthings/zwave-basic-window-shade.src/zwave-basic-window-shade.groovy

```diff
@@ -58,11 +58,26 @@ metadata {
 				defaultValue: false,
 				displayDuringSetup: false
 			)
+            		
+			input(title: "Aeotec Nano Shutter settings",
+			    description: "Sets the Open/Close time.",
```

Comment:
I think having this description up here is redundant. You should probably just make this something like

`"The setting below applies only to the Aeotec Nano Shutter"`